install-dependencies.sh

```diff
@@ -103,6 +92,7 @@ fedora_packages=(
     fakeroot
     file
     dpkg-dev
+    $(cat ./python3-dependencies.txt)
```

Comment:
This breaks tools/toolchain/Dockerfile:

```
ADD ./install-dependencies.sh ./
ADD ./seastar/install-dependencies.sh ./seastar/
```

Now we don't have that file.

In order to avoid splitting the information across many files, you can keep the variable in install-dependencies.sh, and add a flag to extract it: `./install-dependencies --print-python3-runtime-packages` to extract just that list.